tests/test_utils.py

```diff
@@ -29,6 +48,52 @@
     '</video>'
     '</div>'
 )
+QUESTION1_CONTENT = (
+    '<div>'
+    '<p>'
+    'What is the capital of Brazil'
+    '</p>'
+    '<div>'
+    '<img alt="A picture of a map of Brazil" height="71" role="image"'
```

Comment:
Ah, I think I see why you were getting the assert errors. Probably here you can add a space to the end of the string or one to the beginning of the next so the attributes are properly spaced in the final string. That will allow you to remove `squish_list()` in combination with a test change (see corresponding comment below)